### PR TITLE
Aspect ratio

### PR DESCRIPTION
**Pull request by hpcowley**
Added support to specify whether an image should be rendered square or according to natural aspect ratio. For anisotropic EM data, for instance, we should specify that we would like it rendered square. Other users may want the natural aspect ratio.

### CODE REVIEW

**Comment by hpcowley** [comment]
Getting circle CI error about too many local variables in `imshow`. Thinking I could pair it down by having one parameter, `position`, that encapsulates both of our current `position` and `rotation` params.

ie: `position = {'posx' = 0, 'posy' = 0, 'posz' = 0, 'rotx' = Math.pi/2, 'roty' = 0, 'rotz' = 0}`

Although, I admit this is a little wordy for our users... Thoughts? @j6k4m8

**Comment by j6k4m8** [comment]
Definitely would rather just remove that check from our pylint, or disable
for this class specifically. Way better than making weird unintuitive
changes to the code itself

File: pytri/__init__.py [hunk 1]
```diff
@@ -290,11 +292,32 @@ def imshow(
         width = data.shape[1]
         height = data.shape[0]
 
+        if square is True:
```
Comment (j6k4m8) [inline pytri/__init__.py]: This can just be `if square:`

File: pytri/__init__.py [hunk 2]
```diff
@@ -290,11 +292,32 @@ def imshow(
...
+            ar = float(width / height)
+            if ar < 1:
+                width = width * ar
```
Comment (j6k4m8) [inline pytri/__init__.py]: `width *= ar`

File: pytri/__init__.py [hunk 3]
```diff
@@ -290,11 +292,32 @@ def imshow(
...
+            elif ar > 1:
+                height = height * ar
+        if width < scale:
+            width = width * scale
```
Comment (j6k4m8) [inline pytri/__init__.py]: width *= scale

**Pull request closed without merging**